### ISSUE TITLE
#6064 [liqd.net] second-level navigation Accessibility 

### ISSUE THREAD

**Issue by mcastro-lqd** [liqd/liqd-site#146]
**URL:** https://dev.liqd.net/en/academy-landing-page/
**user:** *all*
**important screensize:** *Desktop*
**device & browser:** *Chrome Version 104.0.5112.101 (Official Build) (x86_64) + Pericles Extension*
**Comment/Question:** *I don't know if it is a problem with my screen reader, but the menu is not readable, do you know why?*

<img width="1099" alt="Screenshot 2022-09-08 at 12 01 25" src="https://user-images.githubusercontent.com/59610786/189094860-92cf6e3f-ede4-42c6-a951-fbacffb61495.png">

**Comment by phillimorland** [comment]
@hello-kathi @mcastro-lqd has this been tested with JAWS, NVDA with our testing account or Voiceover? I don't know Pericles Extension and it's not well supported on firefox and their website is outdated, would prefer to know if it works on the most used screen readers

**Comment by hello-kathi** [comment]
@phillimorland I tested it with another one and it worked. So if it fine for @mcastro-lqd I would like to close it.
